crates/ruff_server/README.md

```diff
@@ -9,8 +9,11 @@ files in your editor's workspace, and will refresh its in-memory configuration w
 
 We have specific setup instructions depending on your editor. If you don't see your editor on this list and would like a setup guide, please open an issue.
 
-- Visual Studio Code: Install the [Ruff extension from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff). The language server used by the extension will be, by default, the one in your actively-installed `ruff` binary. If you don't have `ruff` installed and haven't provided a path to the extension, it comes with a bundled `ruff` version that it will use instead. Since the new Ruff language server has not yet been stabilized, you will need to use the pre-release version of the extension and enable the `Experimental Server` setting.
+- Visual Studio Code: Install the [Ruff extension from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff). **As this server is still in beta, you will need to enable the `Native Server` extension setting**. The language server used by the extension will be, by default, the one in your actively-installed `ruff` binary. If you don't have `ruff` installed and haven't provided a path to the extension, it comes with a bundled `ruff` version that it will use instead.
```

Comment:
You should include a JSON snippet demonstrating the exact setting that you need to set, and to what value. People will ignore the text and look to copy-paste.